modules/exploits/linux/http/nuuo_nvrmini_upgrade_rce.rb

```diff
@@ -17,8 +17,8 @@ def initialize(info={})
       'License'        => MSF_LICENSE,
       'Author'         =>
         [
-          'Berk Düşünür <@berkdusunur>', 
-          'numan turle <@numanturle>' 
+          ['Berk Dusunur <@berkdusunur>'], 
```

Comment:
You should be able to use URF8 `Düşünür`.

The `[ ]` aren't required.

Reply:
I solved the problem = []